docs/snippets/common/mdx-canvas-multiple-stories.mdx.mdx

```diff
@@ -1,4 +1,8 @@
 ```md
+<!-- MyComponent.stories.mdx --->
+
+<!--- This is your Story template function, shown here in React -->
```

Comment:
Can we put this above the `export const Template = ` line?

Reply:
Sure, i'll update it shortly for all the snippets.

Reply:
Updated with the latest commit